ports/qemu-riscv/virt.ld

```diff
@@ -0,0 +1,49 @@
+/*
+
+Adapted from https://wiki.osdev.org/RISC-V_Bare_Bones#linker.ld
+
+According to the page history, this content was added after 20110703, and
```

Comment:
Similar comment to the entry.s file: can this be rewritten from scratch?  Ideally following the style of the .ld files in qemu-arm.

Reply:
No problem, I'm testing a new version of this and I'll push the whole lot when it works properly.